src/Findologic/Api/FindologicSearchService.php

```diff
@@ -213,4 +203,52 @@ protected function isCategoryPage(NavigationRequestHandler $handler, ProductList
 
         return !empty($isCategoryPage);
     }
+
+    public function doFilter(ProductListingCriteriaEvent $event): void
+    {
+        $limit = self::FILTER_REQUEST_LIMIT;
```

Comment:
`$limit` is only used here. Instead use the constant and assign it directly to the method in `handleSelectableFilters`:
`$this->handleSelectableFilters($event, $handler, self::FILTER_REQUEST_LIMIT)`